pkg/replica/server.go

```diff
@@ -343,3 +357,79 @@ func (s *Server) PingResponse() error {
 	}
 	return nil
 }
+
+func (s *Server) ShowProfiler() (string, bool, error) {
+	s.profilerLock.Lock()
+	defer s.profilerLock.Unlock()
+
+	logrus.Infof("Prepare to do ShowProfilerConfig")
+	if s.profilerServer != nil {
+		return s.profilerServer.Addr, true, nil
+	}
+	return "", true, nil
+}
+
+func (s *Server) EnableProfiler(addr string, portNumber int32) (string, bool, error) {
```

Comment:
The boolean output is unnecessary. It will be `false` when there is an error. And it's not used in the client side.

Reply:
Good catch!
I add this result at first, indeed it is not used anymore.
Updated!